include/roaring/bitset/bitset.h

```diff
@@ -184,7 +184,7 @@ size_t bitset_symmetric_difference_count(const bitset_t *CBITSET_RESTRICT b1,
     //.....
   }
   */
-inline bool bitset_next_set_bit(const bitset_t *bitset, size_t *i) {
+static inline bool bitset_next_set_bit(const bitset_t *bitset, size_t *i) {
```

Comment:
Making functions `static` makes making bindings in other languages harder, is there a particular reason to prefer `static inline` here?

Reply:
if bindings need them, then not